helmfile/01-applications.yaml

```diff
@@ -106,26 +106,6 @@ releases:
   - values/kube-prometheus-stack-wc.yaml.gotmpl
 {{ end }}
 
-
-- name: ck8sdash
-  namespace: ck8sdash
-  labels:
-    app: ck8sdash
-  chart: ./charts/ck8sdash
-  version: 0.3.1
-  installed: {{ .Values.ck8sdash.enabled }}
```

Comment:
Should we keep this and hard code to `installed: false` to make it auto removed from all environments?

Reply:
Maybe, I wasn't sure whether to do that or write a script that removes everything including the ck8sdash namespace (but maybe that is installed from the rbac chart in most cases).

Reply:
Right, whatever is easier works for me. :slightly_smiling_face:

Reply:
I went with your solution.